addons/locales.sh

```diff
@@ -5,11 +5,8 @@
 # shellcheck disable=2034,2059,1091
 true
 SCRIPT_NAME="Locales"
-
 # shellcheck source=lib.sh
-source /var/scripts/fetch_lib.sh &>/dev/null || . <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null
-
-###################################
+source /var/scripts/fetch_lib.sh &>/dev/null || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null
```

Comment:
3.  I want to see if `source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null` works, too (travis)

Reply:
It seems to work, too! 🚀

Reply:
We need to change a´something else, too:
it must be: `source /var/scripts/fetch_lib.sh || source <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh) &>/dev/null`
else, we will not get any print_text_in_color from fetch_lib.sh (with the disadvantage, that we will see an error message, if the file isn't there. But this is only cosmetic since we want to switch to it later on everywhere.)

Reply:
Whut, so some functions doesn't work if you put it in /dev/null? Then don't do that at all. Harder to debug as well.

Reply:
No, only things that get printed out during the loading of the file. Functions work just fine and aren't affected.

Reply:
If we just load the lib, this doesn't have any disadvantage